packages/nextjs/app/builders/_components/Batch4AddressDisplay.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from "next/link";
+import { Batch4Address } from "./Batch4Address";
+import { Batch4Avatar } from "./Batch4Avatar";
+import { useTheme } from "next-themes";
+import { GlobeAltIcon } from "@heroicons/react/24/solid";
+
+const Batch4AddressDisplay = ({ address, index }: { address: string; index: number }) => {
+  const { resolvedTheme } = useTheme();
+  const isDarkMode = resolvedTheme === "dark";
+
+  return (
+    <div className={`flex flex-col items-center justify-center rounded-lg p-5 bg-white shadow-md`}>
```

Comment:
You don't need the `{` in for the classNames, since you are not doing anything dynamic.

It can be just `className="..."`

Reply:
You can also "fix it" in other places you have the same